src/Integration.Vsix/Resources/Strings.resx

```diff
@@ -277,4 +277,16 @@
   <data name="TextBufferIssueTracker_ProjectGuidError" xml:space="preserve">
     <value>Failed to calculate project guid of file {0}, error occurred: {1}</value>
   </data>
+  <data name="SaveSharedConnectionCommand_SaveSuccess_Message" xml:space="preserve">
+    <value>SonarLint saved the binding configuration to .sonarlint folder in Git repo root. Please commit this file to share it with other contributors. See documentation for more info.</value>
```

Comment:
Not true for all cases (generation should work without git if we have the folder already, shouldn't it?

Reply:
Also, maybe split into 2 paragraphs so it's more readable